<ui version="4.0" >
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Filter Transactions</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="topLabel" >
     <property name="text" >
      <string>Only keep the transactions that meet the following criteria:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="dateBetweenLabel" >
          <property name="text" >
           <string>&amp;Date between</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>fromDateEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2" >
         <widget class="QDateEdit" name="fromDateEdit" />
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="dateAndLabel" >
          <property name="text" >
           <string>and</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <widget class="QDateEdit" name="toDateEdit" >
          <property name="date" >
           <date>
            <year>2999</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="timeBetweenLabel" >
          <property name="text" >
           <string>&amp;Time between</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>fromTimeEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2" >
         <widget class="QTimeEdit" name="fromTimeEdit" />
        </item>
        <item row="1" column="3" >
         <widget class="QLabel" name="timeAndLabel" >
          <property name="text" >
           <string>and</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="4" >
         <widget class="QTimeEdit" name="toTimeEdit" >
          <property name="time" >
           <time>
            <hour>23</hour>
            <minute>59</minute>
            <second>59</second>
           </time>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="pumpLabel" >
          <property name="text" >
           <string>&amp;Pump:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>pumpComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QComboBox" name="pumpComboBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text" >
            <string>Any</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="userLabel" >
          <property name="text" >
           <string>&amp;User:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>userComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QComboBox" name="userComboBox" >
          <item>
           <property name="text" >
            <string>Any</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="companyLabel" >
          <property name="text" >
           <string>&amp;Company:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>companyComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QComboBox" name="companyComboBox" >
          <item>
           <property name="text" >
            <string>Any</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="quantityBetweenLabel" >
          <property name="text" >
           <string>&amp;Quantity between</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>fromQuantitySpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2" >
         <widget class="QDoubleSpinBox" name="fromQuantitySpinBox" >
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="3" >
         <widget class="QLabel" name="quantityAndLabel" >
          <property name="text" >
           <string>and</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="4" >
         <widget class="QDoubleSpinBox" name="toQuantitySpinBox" >
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>100000.000000000000000</double>
          </property>
          <property name="value" >
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="statusLabel" >
          <property name="text" >
           <string>&amp;Status:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>statusComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QComboBox" name="statusComboBox" >
          <item>
           <property name="text" >
            <string>Any</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
